impl/src/main/java/org/ehcache/impl/internal/store/tiering/CacheStore.java

```diff
@@ -336,48 +345,126 @@ public void clear() throws CacheAccessException {
     return cachingTierRef.get();
   }
 
+  @ServiceDependencies({CompoundCachingTier.Provider.class,
+      OnHeapStore.Provider.class, OffHeapStore.Provider.class, OffHeapDiskStore.Provider.class})
   public static class Provider implements Store.Provider {
 
+    private static final Set<Set<ResourceType.Core>> SUPPORTED_RESOURCE_COMBINATIONS;
+    static {
+      // Logic in setTierConfigurations must mirror this set
+      final Set<Set<ResourceType.Core>> supported = new HashSet<Set<ResourceType.Core>>();
+      supported.add(EnumSet.of(ResourceType.Core.HEAP, ResourceType.Core.DISK));
```

Comment:
You have two levels of static import that could be used here -- at what level would you recommend?